### PR TITLE
sorting for monitors

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://jira.rax.io/browse/MNRVA-613

**Description:**

1) Added functionality for sort monitors in which you can sort by ascending and descending order.

**Testing:**

`npm run test:unit`

 ## Screenshots:
sort_monitors

### CODE REVIEW

File: src/app/_features/monitors/components/list/monitorslist.component.ts [hunk 1]
```diff
@@ -26,9 +26,12 @@ export class MonitorslistComponent implements OnInit {
   failedMonitors:any = [];
   total: number;
   page: number = 0;
+  sorting: string = "";
```
Comment (FarajiA) [inline src/app/_features/monitors/components/list/monitorslist.component.ts:29]: Let's add tests for these default values.

File: src/app/_features/monitors/components/list/monitorslist.component.ts [hunk 2]
```diff
@@ -26,9 +26,12 @@ export class MonitorslistComponent implements OnInit {
...
   successCount: number = 0;
   progressVal: number = 0;
   disableOk: boolean = true;
+  isDescending:boolean = true;
```
Comment (FarajiA) [inline src/app/_features/monitors/components/list/monitorslist.component.ts:33]: Same for these two

File: src/app/_features/monitors/components/list/monitorslist.component.ts [hunk 3]
```diff
@@ -128,6 +133,22 @@ export class MonitorslistComponent implements OnInit {
...
   }
 
+  /**
+   * @description sort monitors used for sorting the monitor list by passing params this.sorting.
+   * @param orderBy string
+   * @param sortBy string
+  */  
+  sortMonitors(orderBy, sortBy) {
+    this.isDescending = !this.isDescending;
+    if(this.isSearching === true) {
```
Comment (FarajiA) [inline src/app/_features/monitors/components/list/monitorslist.component.ts:143]: `if (this.isSearching)` right?

**Review by FarajiA — APPROVED** [review]
~~Can you fix the screenshot as well?~~ no worries I updated this.

File: src/app/_features/monitors/components/list/monitorslist.component.ts (hunk 3 above)
Comment (lovedeep92) [reply src/app/_features/monitors/components/list/monitorslist.component.ts:143]: yes .. changed !!

File: src/app/_features/monitors/components/list/monitorslist.component.ts (hunk 1 above)
Comment (lovedeep92) [reply src/app/_features/monitors/components/list/monitorslist.component.ts:29]: yup .. added!

File: src/app/_features/monitors/components/list/monitorslist.component.ts (hunk 2 above)
Comment (lovedeep92) [reply src/app/_features/monitors/components/list/monitorslist.component.ts:33]: done!

**Pull request merged**